lib/firebase_options.dart

```diff
@@ -0,0 +1,58 @@
+// File generated by FlutterFire CLI.
+// ignore_for_file: lines_longer_than_80_chars, avoid_classes_with_only_static_members
+import 'package:firebase_core/firebase_core.dart' show FirebaseOptions;
+import 'package:flutter/foundation.dart'
+    show defaultTargetPlatform, kIsWeb, TargetPlatform;
+
+class DefaultFirebaseOptions {
+  static FirebaseOptions get currentPlatform {
+    if (kIsWeb) {
+      throw UnsupportedError(
+        'DefaultFirebaseOptions have not been configured for web - '
+        'you can reconfigure this by running the FlutterFire CLI again.',
+      );
+    }
+    switch (defaultTargetPlatform) {
+      case TargetPlatform.android:
+        return android;
+      case TargetPlatform.iOS:
+        return ios;
+      case TargetPlatform.macOS:
+        throw UnsupportedError(
+          'DefaultFirebaseOptions have not been configured for macos - '
+          'you can reconfigure this by running the FlutterFire CLI again.',
+        );
+      case TargetPlatform.windows:
+        throw UnsupportedError(
+          'DefaultFirebaseOptions have not been configured for windows - '
+          'you can reconfigure this by running the FlutterFire CLI again.',
+        );
+      case TargetPlatform.linux:
+        throw UnsupportedError(
+          'DefaultFirebaseOptions have not been configured for linux - '
+          'you can reconfigure this by running the FlutterFire CLI again.',
+        );
+      default:
+        throw UnsupportedError(
+          'DefaultFirebaseOptions are not supported for this platform.',
+        );
+    }
+  }
+
+  static const FirebaseOptions android = FirebaseOptions(
+    apiKey: 'AIzaSyCCbpYL8mGCkRD0CZG7bv5WmaNZP67Yxto',
+    appId: '1:1098592064630:android:fa116290547c8adf03fb57',
+    messagingSenderId: '1098592064630',
+    projectId: 'ipraktikum-v1',
+    storageBucket: 'ipraktikum-v1.appspot.com',
+  );
+
+  static const FirebaseOptions ios = FirebaseOptions(
+    apiKey: 'AIzaSyDx6M88HdUC4YfxZK8A4UxwM-AucYrpCe8',
+    appId: '1:1098592064630:ios:0e47f83fa095cedb03fb57',
+    messagingSenderId: '1098592064630',
+    projectId: 'ipraktikum-v1',
+    storageBucket: 'ipraktikum-v1.appspot.com',
+    iosBundleId: 'de.tum.cit.ase.ios2324.tum',
```

Comment:
It s better to expose no secrets, even if they don t work.

Add a boilerPlate code instead to be modified + this [link](https://gist.github.com/carlobortolan/845141e175c2f43135b3893670f99c59) and a short explanation in readme

Reply:
Yes; do .env files work/make sense in flutter? If yes than we can simply put everything there - otherwise I'd leave it as it is for the time being.

Reply:
Yeah we can do that later.

It can be a place holder for now then.